test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -248,6 +249,40 @@ def wrap_obj(obj):
     return wrap_obj
 
 
+def testparams(**kwargs):
+    """
+    Decorator to set or modify test specific parameters
+    """
+    def decorated(function):
+        function.__doc__ += "Test Params:" + "\n\t"
+        for key, value in kwargs.iteritems():
+            function.__doc__ += str(key) + ' : ' + str(value) + '\n\t'
+
+        def wrapper(self, *args):
+            for key, value in kwargs.iteritems():
+                keyname = str(self.__class__.__name__) + "." + key
```

Comment:
I think it's more pythonic to not use two double underscore variables. Instead, you can use `type(self).__name__`
See https://stackoverflow.com/questions/510972/getting-the-class-name-of-an-instance